2018/conference.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: page
+order: 2
+title: "2018"
+subtitle: "Conference"
+titleDisplay: "Conference"
+excerpt:
+venueId:  
+date:
+startTime:
```

Comment:
Best way to capture timing details? venue? Imagining data for the schedule and program, potentially venues <-- but a venue is just for a single event (page) so maybe better handled in front matter